docs/sources/configure-client/grafana-agent/sampling.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Sampling scrape targets"
+menuTitle: "Sampling targets"
+description: "Sampling scrape targets with the Grafana Agent"
+weight: 30
+---
+
+# Sampling scrape targets
+
+It's not uncommon for an application to have many instances deployed. While Pyroscope is designed specifically to handle large amounts of profiling data, you may have other considerations which demand that only a subset of the application's instances should be scraped. For example, the sheer volume of profiling data your application generates may make it unreasonable to profile every instance or you might be targeting cost-reduction. Whatever the case may be, it's possible to sample scrape targets through configuration of the Grafana Agent.
```

Comment:
What do you think about this? I'm not crazy about the last sentence in this suggestion. It still sounds awkward. 
```suggestion
Applications often have many instances deployed.
While Pyroscope is designed to handle large amounts of profiling data, you may want only a subset of the application's instances to be scraped.
For example, the volume of profiling data your application generates may make it unreasonable to profile every instance or you might be targeting cost-reduction.
By configuring Grafana Agent, Pyroscope can sample scrape targets through the Grafana Agent's configuration.
```

Reply:
I also like this updated version -- better for skimming. I do plan to do a video on this at some point and would use the initial version for that

Reply:
I added this change in In https://github.com/grafana/pyroscope/pull/2854/commits/18daf624396b43aa46699122228e2f2edee0e7e1. I tweaked the last sentence a little because saying "Grafana Agent" twice felt awkward.